roles/rails/tasks/main.yml

```diff
@@ -60,7 +60,8 @@
     replace: '{{ env }}:\n  # secret_key_base: ""\n  server_name: "{{ server_hostname }}"'
 
 - name: Generate secret key
-  shell: "source {{ home_dir }}/.rvm/scripts/rvm && bin/rake secret RAILS_ENV={{ env }}"
+  shell: "{{ rvm_command }} && {{ fnm_command }} && bin/rake secret RAILS_ENV={{ env }}"
+
```

Comment:
@Senen It looks like we've accidentally added a blank line here. Sorry I didn't notice before! :pray:

Reply:
Good catch!